package.json

```diff
@@ -46,9 +46,9 @@
   },
   "scripts": {
     "preinstall": "node -e 'process.exit(0)'",
-    "install": "node-pre-gyp install --fallback-to-build",
-    "rebuild": "node-pre-gyp rebuild",
+    "install": "./node_modules/.bin/node-pre-gyp install --fallback-to-build",
```

Comment:
`./node_modules/.bin` should be in the PATH when executing npm scripts using `npm run` -- what issue are you seeing that this solves?